docs/source/tutorial.rst

```diff
@@ -76,6 +77,7 @@ Let's check what tender registry contains:
    :code:
 
 And indeed we have 2 tenders now.
+If you want to retrieve only test tenders you can add `?mode=test` option to the URI. Real and test tenders can be requested if `?mode=_all_` option is specified.
```

Comment:
If you want to retrieve only test tenders, you can add `?mode=test` option to the URI. Both real and test tenders can be requested if `?mode=_all_` option is specified.